### PR TITLE
Quick start guide update

### PR DESCRIPTION
**Pull request by JakaMohorkoDS**
- Reduce quick start guide complexity
- Slight modifications to reference device module to facilitate guide

### CODE REVIEW

File: docs/Antora/modules/getting_started/pages/quick_start_application.adoc [hunk 1]
```diff
@@ -621,44 +432,17 @@ Cpp::
...
-#include <chrono>
-#include <thread>
-
 int main(int argc, const char* argv[])
 {
-    ...
+    // ...
 
-    // Get the first device channel
+    // Get the first signal of the first device's channel
```
Comment (aljazfrancic) [inline docs/Antora/modules/getting_started/pages/quick_start_application.adoc:439]: The first letters of the words "signal" and "device" should perhaps be capitalized throughout the docs, so as to keep the style uniform

File: docs/Antora/modules/getting_started/pages/quick_start_application.adoc [hunk 2]
```diff
@@ -670,68 +454,39 @@ Python::
...
 ====
 
-Reading last value of a signal can be simplified by using the signal's last-value method. The following example shows how to read the last value of the signal.
+==== Packets and Readers
+
+The SDK uses "Packets" to send data through Signals to all listeners. To act as a listener, a Connection with a Signal must be formed which is done by connecting it to an Input Port.
+
+To ease reading data sent by Signals, openDAQ(TM) defines a set of Readers. Readers create an Input Port to which a given Signal is connected. They provide helper methods to ease reading any data that arrives through the formed Connection.
+
+One such Reader is the Stream reader. It presents Packets that arrive through the Connection as a stream of data, abstracting away the concept of Packets from the user. In the example below we create such a Reader that interprets the data sent by the reference Device as a stream of `double` type values. We read up to `100` samples approximately every `25ms`.
```
Comment (aljazfrancic) [inline docs/Antora/modules/getting_started/pages/quick_start_application.adoc:489]: `25 ms` instead of `25ms`

**Review by aljazfrancic — APPROVED** [review]
Nice cleanup!

File: docs/Antora/modules/getting_started/pages/quick_start_application.adoc (hunk 1 above)
Comment (JakaMohorkoDS) [reply docs/Antora/modules/getting_started/pages/quick_start_application.adoc:439]: I think I might keep that for a separate PR, as we're somewhat inconsistent on this throughout the whole documentation. I'll maybe go and actually change everything to lowercase, it's hard to define what should be capitalized and what not (currently it's a bit arbitrary).

**Pull request merged**